OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java

```diff
@@ -67,9 +67,11 @@ public static File getSelectedFile(String extension, String description, DialogT
         int returnVal = 0;
         switch (type) {
             case SAVE:
+                chooser.setDialogTitle("Save As");
```

Comment:
The behavior of the SAVE and SAVEAS cases is different in the getFileWithConfirmation method.  If the type is SAVEAS, the getSelectedFile method is always called.  If the type is SAVE, the getSelectedFile method is only called when currentFile is null.  


```
      public static File getFileWithConfirmation(File currentFile, String extension, String description, DialogType type, boolean fixExtension) {
        File file = currentFile;
        if (file == null || type == DialogType.SAVEAS) {
            file = getSelectedFile(extension, description, type, fixExtension);
        }
        
        file = confirmFileSave(file, currentFile);
        
        return file;
}
```